res/css/views/messages/_ReactionsRowButton.scss

```diff
@@ -33,6 +33,10 @@ limitations under the License.
         background-color: $reaction-row-button-selected-bg-color;
         border-color: $reaction-row-button-selected-border-color;
     }
+    &.mx_ReactionsRowButton_failed {
+        background-color: rgba(140, 12, 12, 0.748);
+        border-color: rgba(68, 0, 0, 0.748);
```

Comment:
Those two colours should probably be stored in a variable as we might want to use a slightly different shade for contrast depending on the theme used

Reply:
Done.